tests/bench/fixtures/fixtures.go

```diff
@@ -104,7 +128,16 @@ func ExtractGQLFromType(t any) (string, error) {
 		fname := f.Name
 		ftype := f.Type.Name()
 		gqlType := gTypeToGQLType[ftype]
-		fmt.Fprintf(&buf, "\t%s: %s\n", fname, gqlType)
+
+		directive := ""
+		if g.options != nil {
+			if dirsMap, ok := g.options.directives[name]; ok {
+				if dirs, ok := dirsMap[fname]; ok {
+					directive = " " + strings.Join(dirs, " ")
+				}
+			}
+		}
+		fmt.Fprintf(&buf, "\t%s: %s%s\n", fname, gqlType, directive)
```

Comment:
todo: Please document this block, and maybe rename `directive` to `directives`, as it appears to hold multiple.